src/ui/osx/TogglDesktop/TimeEntryViewItem.m

```diff
@@ -136,7 +136,7 @@ - (void)load:(TogglTimeEntryView *)te
 		self.Error = nil;
 	}
 	// If duration is less than 15 seconds delete without confirmation
-	self.confirmlessDelete = (self.duration_in_seconds < 15);
+	self.confirmlessDelete = (self.duration_in_seconds > -15 || self.duration_in_seconds < 15);
```

Comment:
Can you elaborate why it must be greater than `-15`?

Reply:
whoops, this is actually wrong :D